src/services/services.ts

```diff
@@ -3987,7 +3987,7 @@ module ts {
 

```

Comment:
that does not work as getCanonicalFileName changes the case. which seems the right thing to do if you are comparing two paths on a case-insensitive system